src/robots_config/launch/audio.launch~

```diff
@@ -0,0 +1,4 @@
+<!-- -*- mode: XML -*- -->
+<launch>
+    <node name="AudioFeature" pkg="audiosysneeds" type="audioSource.py" output="screen"></node>
+</launch>
```

Comment:
Temporary file shouldn't check in. 

I think you have a lot of useless files checked in. Before you ask me review it, please first check yourself if the PR is ready for merge. Basically you should NOT check in those irrelevant or temporary files. You can edit/create .gitignore file if needed to tell git to ignore those files.